src/app/modules/main/wallet_section/transactions/model.nim

```diff
@@ -169,14 +222,35 @@ QtObject:
         t.value,
         t.fromAddress,
         t.to,
-        t.contract
+        t.contract,
+        t.chainId,
+        t.maxFeePerGas,
+        t.maxPriorityFeePerGas,
+        t.input,
+        t.txHash,
+        t.multiTransactionID,
+        false,
+        t.baseGasFees,
+        self.getTotalFees(t.maxPriorityFeePerGas, t.baseGasFees, t.gasUsed, t.maxFeePerGas),
```

Comment:
With previous comment here we would only do: `t.totalFees()` all parameters are from the t object so that could be encapsulated into this type.